.github/workflows/simple-ci.yml

```diff
@@ -0,0 +1,67 @@
+# This is a basic workflow
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
+
+
+name: CI-pipeline
+env:
+  APP_NAME    : "converter"
+  PACKAGE_NAME: "converter_app_${{ github.sha }}"
+
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    branches: [ "main" ]
+  push:
+    branches-ignore: [ "main", "dev" ]
+
+jobs:
+  build-app:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
@MariaMaximova, I think we can consider splitting steps in different jobs by the target: 1. code style; 2. then unit tests; 3. other tests (functional, etc.). Also we can block further jobs if previous one is failed (e.g. code style is failed, no need to run unittests then, because the MR will be updated anywhere with code-style fixes.
You can resolve this comment because it is as a TODO for the future.

Reply:
Could you please check this commit? [2dea5cd](https://github.com/MariaMaximova/convert_temp/pull/11/commits/2dea5cd663eba79601b6a65c1c672924d283e663)

Reply:
@OleksiiBulba Could you please check this commit? [2dea5cd](https://github.com/MariaMaximova/convert_temp/pull/11/commits/2dea5cd663eba79601b6a65c1c672924d283e663)